.github/workflows/pull-build-images.yaml

```diff
@@ -80,4 +80,12 @@ jobs:
          name: kyma-environment-globalaccounts
          dockerfile: Dockerfile.globalaccounts
          context: .
-         build-args: BIN=globalaccounts
\ No newline at end of file
+         build-args: BIN=globalaccounts
+
+   schema-migrator-image:
+      uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+      with:
+         name: kyma-environment-broker-schema-migrator
+         dockerfile: Dockerfile.schemamigrator
+         context: .
+         build-args: BIN=schema-migrator
```

Comment:
The schema migrator Dockerfile doesn't use build-args, so this section is not needed.

Reply:
Also don't use full `kyma-environment-broker-schema-migrator` name. Look how other jobs are named.

Reply:
I removed `build-args` input.